2021-06-30T15:32:02|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\FastFMT-micro\anaconda3\lib\site-packages\ScopeFoundry\measurement.py", line 244, in start_stop
    self._start()
  File "C:\Users\FastFMT-micro\anaconda3\lib\site-packages\ScopeFoundry\measurement.py", line 134, in _start
    raise RuntimeError("Cannot start a new measurement while still measuring")
RuntimeError: Cannot start a new measurement while still measuring

2021-06-30T16:02:58|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\FastFMT-micro\anaconda3\lib\site-packages\ScopeFoundry\measurement.py", line 244, in start_stop
    self._start()
  File "C:\Users\FastFMT-micro\anaconda3\lib\site-packages\ScopeFoundry\measurement.py", line 134, in _start
    raise RuntimeError("Cannot start a new measurement while still measuring")
RuntimeError: Cannot start a new measurement while still measuring

2021-06-30T18:46:46|ERROR|hyperMeasure|hyper Failed to update figure1: Timeout error (-7): '@ GCSMessages._read'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 88, in update_display\n    self.stage.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\hardware.py", line 181, in read_from_hardware\n    lq.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\logged_quantity.py", line 161, in read_from_hardware\n    val = self.hardware_read_func()\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PI_GCS_device.py", line 44, in get_servo\n    servomode = self.pi_device.qSVO(self.axis)[self.axis]\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcs2\\gcs2commands.py", line 689, in qSVO\n    answer = self._msgs.read(cmdstr)\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 125, in read\n    answer = self._read(stopon)\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 180, in _read\n    raise GCSError(gcserror.E_7_COM_TIMEOUT, \'@ GCSMessages._read\')\n', "PI_ScopeFoundry.PIPython.pipython.pidevice.gcserror.GCSError: Timeout error (-7): '@ GCSMessages._read'\n"]
2021-06-30T18:46:46|ERROR|hyperMeasure|hyper Failed to update figure1: Controller sent unexpected response (-1004). ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 88, in update_display\n    self.stage.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\hardware.py", line 181, in read_from_hardware\n    lq.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\logged_quantity.py", line 161, in read_from_hardware\n    val = self.hardware_read_func()\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PI_GCS_device.py", line 68, in get_position\n    position = self.pi_device.qPOS(self.axis)[self.axis]\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcs2\\gcs2commands.py", line 667, in qPOS\n    answer = self._msgs.read(cmdstr)\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 143, in read\n    self._checkerror()\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 302, in _checkerror\n    raise exc  # Raising NoneType while only classes or instances are allowed pylint: disable=E0702\n', 'PI_ScopeFoundry.PIPython.pipython.pidevice.gcserror.GCSError: Controller sent unexpected response (-1004)\n']
2021-06-30T18:46:51|ERROR|hyperMeasure|hyper Failed to update figure1: Controller sent unexpected response (-1004). ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 88, in update_display\n    self.stage.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\hardware.py", line 181, in read_from_hardware\n    lq.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\logged_quantity.py", line 161, in read_from_hardware\n    val = self.hardware_read_func()\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PI_GCS_device.py", line 68, in get_position\n    position = self.pi_device.qPOS(self.axis)[self.axis]\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcs2\\gcs2commands.py", line 667, in qPOS\n    answer = self._msgs.read(cmdstr)\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 143, in read\n    self._checkerror()\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 302, in _checkerror\n    raise exc  # Raising NoneType while only classes or instances are allowed pylint: disable=E0702\n', 'PI_ScopeFoundry.PIPython.pipython.pidevice.gcserror.GCSError: Controller sent unexpected response (-1004)\n']
2021-06-30T18:47:24|ERROR|hyperMeasure|hyper Failed to update figure1: Controller sent unexpected response (-1004). ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 88, in update_display\n    self.stage.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\hardware.py", line 181, in read_from_hardware\n    lq.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\logged_quantity.py", line 161, in read_from_hardware\n    val = self.hardware_read_func()\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PI_GCS_device.py", line 68, in get_position\n    position = self.pi_device.qPOS(self.axis)[self.axis]\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcs2\\gcs2commands.py", line 667, in qPOS\n    answer = self._msgs.read(cmdstr)\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 143, in read\n    self._checkerror()\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 302, in _checkerror\n    raise exc  # Raising NoneType while only classes or instances are allowed pylint: disable=E0702\n', 'PI_ScopeFoundry.PIPython.pipython.pidevice.gcserror.GCSError: Controller sent unexpected response (-1004)\n']
2021-06-30T18:48:26|ERROR|hyperMeasure|hyper Failed to update figure1: Controller sent unexpected response (-1004). ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 88, in update_display\n    self.stage.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\hardware.py", line 181, in read_from_hardware\n    lq.read_from_hardware()\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\logged_quantity.py", line 161, in read_from_hardware\n    val = self.hardware_read_func()\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PI_GCS_device.py", line 68, in get_position\n    position = self.pi_device.qPOS(self.axis)[self.axis]\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcs2\\gcs2commands.py", line 667, in qPOS\n    answer = self._msgs.read(cmdstr)\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 143, in read\n    self._checkerror()\n', '  File "C:\\LabPrograms\\Python\\PI_ScopeFoundry\\PIPython\\pipython\\pidevice\\gcsmessages.py", line 302, in _checkerror\n    raise exc  # Raising NoneType while only classes or instances are allowed pylint: disable=E0702\n', 'PI_ScopeFoundry.PIPython.pipython.pidevice.gcserror.GCSError: Controller sent unexpected response (-1004)\n']
2021-07-02T15:44:12|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:13|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:13|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:13|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:13|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:13|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:13|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:14|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:14|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:14|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:14|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:15|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:15|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:15|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:15|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:15|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:16|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:16|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:16|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:16|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:16|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:17|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:17|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:17|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:17|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:17|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:18|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:18|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:18|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:18|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:18|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:19|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:19|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:19|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:19|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:19|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:20|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:20|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:20|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:20|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:20|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:21|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:21|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:21|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:21|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:21|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:22|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:22|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:22|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:22|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:22|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:23|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:23|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:23|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:23|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:23|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:24|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:24|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:24|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:24|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:25|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:25|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:25|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:25|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:25|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:26|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:26|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:26|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:26|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:26|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:27|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:27|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:27|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:27|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:27|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:28|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:28|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:28|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:28|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:28|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:29|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:29|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:29|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:29|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:29|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:30|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:30|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:30|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:30|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:30|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:31|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:31|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:31|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:31|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:31|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:32|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:32|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:32|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:32|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:33|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:33|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:33|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:33|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:33|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:34|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:34|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:34|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:34|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:35|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:35|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:35|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:35|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:36|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:36|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:36|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:36|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:36|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:37|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:37|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:37|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:37|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:38|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:38|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:38|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:38|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:38|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:39|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:39|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:39|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:39|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:40|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:40|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:40|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:40|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:40|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:41|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:41|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:41|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:41|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:42|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:42|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:42|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:42|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:43|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:43|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:43|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:43|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:43|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:44|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:44|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:44|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:44|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
2021-07-02T15:44:45|ERROR|hyperMeasure|hyper Failed to update figure1: division by zero. ['Traceback (most recent call last):\n', '  File "C:\\Users\\FastFMT-micro\\anaconda3\\lib\\site-packages\\ScopeFoundry\\measurement.py", line 273, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\LabPrograms\\Python\\HyperspectralMicroscope\\hyperspectral_measure.py", line 94, in update_display\n    self.settings[\'progress\'] = (self.frame_index +1) * 100/length\n', 'ZeroDivisionError: division by zero\n']
